doc/install_generic.txt

```diff
@@ -49,34 +50,15 @@ Bleeding-Edge Installation
 The following are general instructions that will set you up with the
 bleeding-edge version of Theano.
 
-.. code-block:: bash
-
-    git clone git://github.com/Theano/Theano.git
-    pip install [--user] -e git+git:github.com:Theano/Theano.git [--src cloned_folder]
-
-The advantage of doing the -e switch is that it allows pip to keep track of Theano and
-its dependancies. 
-
-If you do not wish to use pip, you can simply run :
-
-.. code-block:: bash
-
-    git clone git://github.com/Theano/Theano.git
-    cd Theano
-    python setup.py develop
-
-Using ``develop`` mode is
-preferable to ``install`` as any modifications you make in the checkout
-directory (or changes you pull with Git) will be automatically reflected
-in the "installed" version without re-running ``python setup.py install``.
+.. raw:: html
 
-If you do not have permission to modify your ``site-packages`` directory you
-can specify an alternative installation prefix using
+    <pre><span class="red">&#60;sudo&#62;</span> pip install <span class="blue">&#60;--user&#62;</span> <span class="green">&#60;-e&#62;</span> git+https://github.com/Theano/Theano.git#egg=Theano <span class="green">&#60;--src clonedfolder&#62;</span></pre>
```

Comment:
`clonedfolder` Should be `source_folder_path` or something like that.
Also this command does not work on 2 computer I've tried.
Finally, I like the idea on just one master command but I think the `clone` + `pip -e` should be there also.